ci-operator/config/openshift/csi-external-resizer/openshift-csi-external-resizer-master.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.11
+canonical_go_repository: github.com/kubernetes-csi/external-resizer
+# No images yet, it's not to be released in OCP yet
```

Comment:
Maybe building those images but not promoting them (deleting the `promotion`) could be more useful in that case?

Reply:
Ok, I commented out promotion, I hope that it does what I think it does :-)

Reply:
It should give you an image-building presubmit (so you know your images build) but not the promoting postsubmit (so no images are kept or promoted anywhere).